tf2onnx/onnx_opset/math.py

```diff
@@ -22,7 +22,7 @@
 
 # pylint: disable=unused-argument,missing-docstring
 
-@tf_op(["Add", "AddV2", "Div", "Mul", "Sub"])
+@tf_op(["Add", "AddV2", "Div", "Mul", "Sub", "LessOrEqual"])
```

Comment:
LessOrEqual belongs into logical.py. Also - we do support his already so you need to be careful how you do this.

Reply:
Yes, apologies.
But for the one in logical.py, shall we map it to the latest LessOrEqual or just leave it as it is.

Reply:
Added opset 12 GreaterOrEqual and LessOrEqual to simplify the logic.